.circleci/config.yml

```diff
@@ -131,11 +132,26 @@ references:
       name: Run rubocop
       command: bundle exec rubocop
 
+  setup_test_reporter: &setup_test_reporter
+    run:
+      name: Setup Code Climate test-reporter
+      command: |
+        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        chmod +x ./cc-test-reporter
+        ./cc-test-reporter before-build
+
   run_unit_and_feature_tests: &run_unit_and_feature_tests
     run:
       name: Run unit and feature tests
       command: |
-        SKIP_SIMPLECOV=true bundle exec rspec
+        SKIP_SIMPLECOV=false bundle exec rspec
```

Comment:
Do we need the SKIP_SIMPLECOV=false?

Reply:
So the idea of having a variable to switch off action of collecting test coverage is not from me, it is designed cross all those ruby apps,  but with different name e.g. i CTS ,it is called as 'COVERAGE', anyway,  IMO,  such idea is ok.  Before this one was switched off.  I switched it on unless there is reason you can recall why it was switch off.  When I switched it on initially, there were couple of bugs, I fixed them.